addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -251,15 +271,28 @@ const FontFamilyPickerUserValueWidget = SelectUserValueWidget.extend({
 
         // Remove Google font
         const googleFontIndex = parseInt(ev.target.dataset.fontIndex);
-        const googleFont = this.googleFonts[googleFontIndex];
-        this.googleFonts.splice(googleFontIndex, 1);
+        const isLocalFont = ev.target.dataset.fontLocal;
+        let googleFontName;
+        if (isLocalFont) {
+            const googleFont = this.googleLocalFonts[googleFontIndex].split(':');
+            googleFontName = googleFont[0];
+            await this._rpc({
+                model: 'web_editor.assets',
+                method: 'unlink_local_google_font',
+                args: [googleFont[1]],
+            });
```

Comment:
A bit sad to make one rpc for the deletion and one for the update of the scss variable, maybe this could be done in one-go? Would also take care of the portal user problem probably?

Reply:
Indeed, might as well do both in one go. Changed, added a `delete-font-attachment-id` entry in the values which is popped later before save.